### PR TITLE
HTBHF-2495 remove apply link on guidance page

### PR DESCRIPTION
**Pull request by gethyn1**

### CODE REVIEW

File: src/web/views/guidance/en/how-it-works.njk [hunk 1]
```diff
@@ -19,7 +19,7 @@
   </ul>
 
   <h2 class="govuk-heading-l">If you’re already on the scheme</h2>
-  <p class="govuk-body">You’ll no longer get milk and food vouchers. You need to <a class="govuk-link" href="/apply">apply for a Healthy Start</a> money card instead.</p>
```
Comment (adeel718) [inline src/web/views/guidance/en/how-it-works.njk]: More of a question - does the acceptance tests go to the level of checking content and clicking the link?

Comment (gethyn1) [reply src/web/views/guidance/en/how-it-works.njk]: Not on the guidance pages (I don't think), but if it is critical to the application we will check that.

Comment (gethyn1) [reply src/web/views/guidance/en/how-it-works.njk]: We check the titles and "previous" and "next" links on each guidance page

Comment (gethyn1) [reply src/web/views/guidance/en/how-it-works.njk]: https://github.com/DepartmentOfHealth-htbhf/htbhf-acceptance-tests/blob/master/src/test/java/uk/gov/dhsc/htbhf/steps/GuidanceSteps.java

Comment (adeel718) [reply src/web/views/guidance/en/how-it-works.njk]: Thanks, I have raised this with Jyothi as we don't check content for some of our services, I think we should continue doing this

**Pull request merged**